lib/opentok/opentok.rb

```diff
@@ -177,8 +177,11 @@ def sip
       @sip ||= Sip.new client
     end
 
-    protected
+    def signal
+      @signal ||= Signals.new client
```

Comment:
Are you missing the require of the "Signals" class (require "opentok/signals") in this file?